### PR TITLE
Vincent http server ip

### PR DESCRIPTION
**Pull request by Vincentmask**
fixed UI tests
fixed server can't get/post across different machine
changed IP address to get the server working across machines

The website needs to fetch a special IP address to interact with the server, couldn't work in previous code when fetching localhost.

Jeff fixed the background color
Moved the title 9 subtitle above the submit button

### CODE REVIEW

**Review by imOnVacation — APPROVED** [review]
Thanks, Jeff and Vincent.

Vincent, can you explain why we are adding the IP address to the server file?

**Review by rsanch2 — APPROVED** [review]

**Review by MrFiszbi1 — APPROVED** [review]

File: src/safeUI/SafeUI.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ function SafeUI() {
 
       // fetch data from API endpoint
       // param: request method, header, body
-      fetch('http://localhost:3001/addMessage', {
+      fetch('http://131.252.208.28:3001/addMessage', {
```
Comment (MrFiszbi1) [inline src/safeUI/SafeUI.tsx]: I am not totally sure, but I am not sure if this is safe. In the full stack class, I am taking we put the host and port IP addresses in a separate .env file. Maybe I am misunderstanding the two things. Not sure, do you know what I am talking about?

**Review by MrFiszbi1 — CHANGES_REQUESTED** [review]

File: src/safeUI/SafeUI.tsx (hunk 1 above)
Comment (NAlexH2) [reply src/safeUI/SafeUI.tsx]: I'm with Jafar on this. This enables various attack vectors.

Comment (Vincentmask) [reply src/safeUI/SafeUI.tsx]: Yeah that was my concert at the beginning as well. I am thinking to have this as part of the script like the database info we had

**Comment by Vincentmask** [comment]
Changed the port back to localhost. Once we got the VM and domain working we can hide the IP address later

**Review by MrFiszbi1 — APPROVED** [review]

**Pull request merged**